go/vt/vtgate/planbuilder/operators/aggregator.go

```diff
@@ -157,6 +167,7 @@ func (a *Aggregator) AddColumn(ctx *plancontext.PlanningContext, reuse bool, gro
 	if reuse {
 		offset := a.findColInternal(ctx, ae, groupBy)
 		if offset >= 0 {
+			a.checkOffset(offset)
```

Comment:
these are in two places here while returning, should we add a defer call instead?